### DISCUSSION TITLE
Entity Linking sample project uses model architecture that is not registered.

### DISCUSSION THREAD

**Discussion by narayanacharya6** [explosion/spaCy#8548] [Help: Other Questions] [answered]
I have been playing around with the Entity Linking tutorial project [here](https://github.com/explosion/projects/tree/v3/tutorials/nel_emerson) by running it on some datasets. 

What I just noticed is that the model architecture for the entity linker in the config [file](https://github.com/explosion/projects/blob/v3/tutorials/nel_emerson/configs/nel.cfg#L50) is `spacy.HashEmbedCNN.v1` but I don't see any model architecture registered with that name in spaCy. There is a `spacy.HashEmbedCNN.v2` which I believe is what should have been in the config file but maybe the project is old and it was built when v1 existed. The main issue is that spaCy training with `spacy.HashEmbedCNN.v1` still works. I would expect this to fail complaining that no such architecture was found but that is not the case. What am I missing here?

**Comment by adrianeboyd** [comment, marked as answer]
The older versions are moved to `spacy-legacy`: https://github.com/explosion/spacy-legacy

**Reply by narayanacharya6** [reply]
I did not realize I had `spacy-legacy` installed in my environment. This makes sense now. Thanks!
